spec/requests/groups_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
- [ ] Great job so far 👏🏻 However according to the project requirements. The App should have unit tests and integration tests for all the most important components of the RoR application. Kindly make sure all the components have `integration` tests as weel ✍🏻